.cirrus.yml

```diff
@@ -133,6 +133,19 @@ integration_task:
   setup_script: *setup
   main_script: *main
 
+podman_e2e_task:
+  alias: "podman_e2e"
+  depends_on:
+    - "unit"
+  env:
+    GOPATH: &gopath "/var/tmp/go"
+    GOPATH: &gocache "/var/tmp/go/cache/go-build"
+  cargo_cache: *ro_cargo_cache
+  targets_cache: *ro_targets_cache
+  bin_cache: *ro_bin_cache
```

Comment:
This will get you a working `/var/tmp/netavark/bin/netavark` binary. 

The latest Aardvark is available by curl & unzip: https://api.cirrus-ci.com/v1/artifact/github/containers/aardvark-dns/success/binary.zip